modules/search/class-jetpack-instant-search.php

```diff
@@ -119,11 +119,12 @@ protected function inject_javascript_options() {
 		$prefix  = Jetpack_Search_Options::OPTION_PREFIX;
 		$options = array(
 			'overlayOptions'        => array(
-				'colorTheme'      => get_option( $prefix . 'color_theme', 'light' ),
-				'enableInfScroll' => (bool) get_option( $prefix . 'inf_scroll', false ),
-				'highlightColor'  => get_option( $prefix . 'highlight_color', '#FFC' ),
-				'opacity'         => (int) get_option( $prefix . 'opacity', 97 ),
-				'showPoweredBy'   => (bool) get_option( $prefix . 'show_powered_by', true ),
+				'colorTheme'       => get_option( $prefix . 'color_theme', 'light' ),
+				'enableInfScroll'  => (bool) get_option( $prefix . 'inf_scroll', false ),
+				'enableTransition' => (bool) get_option( $prefix . 'overlay_transition', false ),
```

Comment:
I'm concerned about adding a configuration setting for something as minute as the overlay transition. 

Wouldn't it be simpler to simply omit this setting entirely and default to having a 150ms transition?

Reply:
Yep, we can do that. I don't think it warrants the customizer option, but wanted to add it here so we could discuss.

Reply:
+100